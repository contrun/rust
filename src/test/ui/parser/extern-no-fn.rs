extern {
    f(); //~ ERROR non-item in item list
}

fn main() {
}
