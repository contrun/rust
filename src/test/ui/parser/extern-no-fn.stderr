error: non-item in item list
  --> $DIR/extern-no-fn.rs:2:5
   |
LL | extern {
   |        - item list starts here
LL |     f();
   |     ^ non-item starts here
LL | }
   | - item list ends here

error: aborting due to previous error

